server/app.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const sampledata = require('../sampledata.js');
+
+// parse application/x-www-form-urlencoded
+app.use(bodyParser.urlencoded())
+
+// parse application/json
+app.use(bodyParser.json())
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+app.get('/feed/global', (req, res) => {
+
+  // DB query should replace default promise
+  Promise.resolve(sampledata.sampleFeed)
+    .then((results) => {
+      res.status(200).json({items: results});
+    })
+    .catch((err) => {
```

Comment:
In the API Spec we defined errors:
Error Response:

Code: 401 UNAUTHORIZED 
Content: { error : "Log in" }

Code: 500 INTERNAL SERVER ERROR 
Content: { error :"server error" }

Reply:
@jGlass314 I don't think this is an appropriate time to add those errors. Those errors are dependent on the DB query response and the cookies from authorization. Since neither of those features are implemented yet, I think a skeleton endpoint with 404 is more appropriate at this time.

Reply:
Discussed the issue. This is stub code and will be changed later.